psi4/driver/procrouting/response/scf_response.py

```diff
@@ -249,3 +254,97 @@ def _print_output(complete_dict, output):
+       as every element. (Convenient for C1 systems)
+    states_per_irrep : list (int), {optional}
+       The solver will find this many lowest excitations for each irreducible representation of the computational point group.
+       The default is to find the lowest excitation of each symmetry. If this option is provided it must have the same number of elements
+       as the number of irreducible representations as the computational point group.
+    triplets : str {optional, ``none``, ``only``, ``also``}
+       The default ``none`` will solve for no triplet states, ``only`` will solve for triplet states only, and ``also``
+       will solve for the requested number of states of both triplet and singlet. This option is only valid for restricted references,
+       and is ignored otherwise. The triplet and singlet solutions are found separately so using the ``also`` option will roughly double
+       the computational cost of the calculation.
+    tda :  bool {optional ``False``}
+       If true the Tamm-Dancoff approximation (TDA) will be employed. For HF references this is equivalent to CIS.
+    e_tol : float, {optional, 1.0e-6}
+       The convergence threshold for the excitation energy
+    r_tol : float, {optional, 1.0e-8}
+       The convergence threshold for the norm of the residual vector
+    max_ss_vectors: int {optional}
+       The maximum number of ss vectors that will be stored before a collapse is done.
+    guess : str
+       If string the guess that will be used. Allowed choices:
+       - ``denominators``: {default} uses orbital energy differences to generate guess vectors.
+       - ``random``: generates random vectors
+
+
+    ..note:: The algorithm employed to solve the non-Hermitian eigenvalue problem
+             (when ``tda`` is False) will fail when the SCF wavefunction has a triplet instability.
+    """
+    # gather arguments
+    etol = kwargs.pop('e_tol', 1.0e-6)
+    rtol = kwargs.pop('r_tol', 1.0e-8)
+    max_ss_vec = kwargs.pop('max_ss_vectors', 50)
+
+    # how many
+    passed_spi = kwargs.pop('states_per_irrep', [])
+    nstates = kwargs.pop('nstates', 1)
+    #states_per_irrep = _validate_states_args(wfn, passed_spi, nstates)
+    states_per_irrep = passed_spi
+
+    # guess
+    passed_guess = kwargs.pop('guess', 'random')
+    guess_type = None
+    if isinstance(passed_guess, (list, tuple)):
+        guess_vectors = _validate_user_guess(wfn, states_per_irrep, guess)
+        guess_type = 'user'
+    else:
+        guess_vectors = []
+        guess_type = 'random'
+
+    # which problem
+    ptype = 'rpa'
+    solve_function = solvers.hamiltonian_solver
+    if kwargs.pop('tda', False):
+        ptype = 'tda'
+        solve_function = solvers.davidson_solver
+
+    # construct the engine
+    if wfn.same_a_b_orbs():
+        engine = TDRSCFEngine(wfn, triplet=kwargs.pop('triplet', False), ptype=ptype)
+    else:
+        engine = TDUSCFEngine(wfn, ptype=ptype)
```

Comment:
Can we print out a banner at this point that displays the TDSCF module and then prints author, settings, and a warning that this is experimental?